.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17.5'
+      - run: make test
+
```

Comment:
travis is currently not working, so I'm hoping to swap to GA

I'm not gonna know if it works until we merge this though